### ISSUE TITLE
API for getting currently focussed/hovered emoji

### ISSUE THREAD

**Issue by rugk** [missive/emoji-mart#444]
Split idea from https://github.com/missive/emoji-mart/pull/431#issuecomment-650393063

TL;DR is: An API for getting the currently focussed/hovered emoji, especially if https://github.com/missive/emoji-mart/pull/431 is merged, would be really useful to adjust/react on more things.

----

**Use case**
[I have a use case](https://github.com/rugk/awesome-emoji-picker/issues/100), where I'd like more shortcuts to trigger something in the picker/with the selected emoji. In my case Ctrl+C should e.g. trigger a copy to clipboard.

**Currently**
As all methods we can use to get the selected/chosen emoji currently are _reactive_ (i.e. as an dev outside the lib you only get the emoji if something happens as as a result of a click or so) this is hard to implement.

**What we want**
A _active_ method that could just return the currently focused emoji could be very helpful for that. (Note this use case barely existed before, because if you click an emoji you get the reaction.)
So e.g. a `getCurrentlyFocussedEmoji` that likely is used internally in a similar way, may be very useful here.

Or… of course… you just expose the current keypress handler. However, IMHO, that would be less flexible as an API, because you are then – yet again – limited to key presses only.

**Edit:** Don't know whether that may also return if an emoji is only hovered hmm… maybe it can do both, possibly in two separate functions. :thinking:
